src/ncp/spinel.h

```diff
@@ -535,6 +535,12 @@ typedef enum
     SPINEL_PROP_THREAD_PREFERRED_ROUTER_ID
                                         = SPINEL_PROP_THREAD_EXT__BEGIN + 10,
 
+    /// Thread Neighbor Table
+    /** Format: `A(ESLLLCcCb)`
```

Comment:
Wrong format. Should be `A(T(ESLLLCcCb))`. `A(ESLLLCcCb)` is a different on the wire.